src/main/resources/application-ebs.yml

```diff
@@ -0,0 +1,35 @@
+spring:
```

Comment:
No need to add all properties to ebs.yml file, this file should only contain the properties that are being overriden